elasticapm/transport/base.py

```diff
@@ -176,11 +176,11 @@ def _process_queue(self):
             if flush:
                 if buffer_written:
                     self._flush(buffer, forced_flush=forced_flush)
-                elif forced_flush and "/localhost:" in self.client.config.server_url:
+                elif forced_flush and any(x in self.client.config.server_url for x in ("/localhost:", "/127.0.0.1:")):
```

Comment:
How much do you care about possible IPv6 usage? :)

Reply:
I don't think we care for this particular instance -- seems likely that most users would use `localhost` -- the only reason I added `127.0.0.1` was because the testing http server uses that as its URL base.

Those using the local APM server should just use our default `server_url`, which is `'http://localhost:8200'`